lib/kubernetes-deploy/kubectl.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+require 'kubeclient'
+
+module KubernetesDeploy
```

Comment:
This extraction should probably be a separate PR and should cover the KubernetesResource version as well: https://github.com/Shopify/kubernetes-deploy/blob/fd557925825a3a672f62a2ff0984bee3bf1b519b/lib/kubernetes-deploy/kubernetes_resource.rb#L115

@kirs tried to do this previously in https://github.com/Shopify/kubernetes-deploy/pull/58, but we disagreed on the implementation. This version seems reasonable to me (although needs changes to be used in the additional location), but we should reopen that discussion separately.